packages/fast-layouts-react/src/column/column.tsx

```diff
@@ -91,7 +92,7 @@ export class Column extends Foundation<ColumnHandledProps, ColumnUnhandledProps,
      */
     private getValueByBreakpoint<T>(breakpointSet: T[]): T {
         if (!canUseDOM()) {
-            return breakpointSet[0];
+            return breakpointSet[this.props.defaultBreakpoint];
```

Comment:
Can you add a test for this configuration? If there isn't one already, would be great to have a test for the default as well.

Reply:
Sure. I will check if there is any test already or not, and I will either update the existing or make a new one.